### ISSUE TITLE
What's the meaning of "yesno" in code?

### ISSUE THREAD

**Issue by Maybewuss** [alontalmor/MultiQA#25]
I don't know the meaning of 'yesno_labels' .
`self.qa_yesno = torch.nn.Linear(self._text_field_embedder.get_output_dim(), 3)`
and why the linear dim is 3?

**Comment by alontalmor** [comment]
Hi, these are yesno questions from HotpotQA, the options are yes, no , not_a_yes_no question, thus the size is 3.

**Comment by Maybewuss** [comment]
> Hi, these are yesno questions from HotpotQA, the options are yes, no , not_a_yes_no question, thus the size is 3.

Thanks.
